repos/system_upgrade/el7toel8/actors/sssdfacts/actor.py

```diff
@@ -0,0 +1,33 @@
+import ConfigParser
+
+from leapp.actors import Actor
+from leapp.libraries.actor.library import SSSDFactsLibrary
+from leapp.models import SSSDConfig
+from leapp.tags import IPUWorkflowTag, FactsPhaseTag
+
+
+class SSSDFacts(Actor):
+    '''
+    Check SSSD configuration for changes in RHEL8 and report them in model.
+
+    These changes are:
+    - id_provider=local is no longer supported and will be ignored
+    - ldap_groups_use_matching_rule_in_chain was removed and will be ignored
+    - ldap_initgroups_use_matching_rule_in_chain was removed and will be ignored
+    - ldap_sudo_include_regexp changed default from true to false
+    '''
+
+    name = 'sssd_facts'
+    consumes = ()
+    produces = (SSSDConfig,)
+    tags = (IPUWorkflowTag, FactsPhaseTag)
+
+    def process(self):
+        try:
+            config = ConfigParser.RawConfigParser()
+            config.read('/etc/sssd/sssd.conf')
+        except ConfigParser.Error:
+            # SSSD is not configured properly. Nothing to do.
+            config = ConfigParser.RawConfigParser()
+
+        self.produce(SSSDFactsLibrary(config).process())
```

Comment:
Does it actually make sense at all to produce this message in the end if you fail to read the SSSD config?

Reply:
If this case is handled correctly in SSSDCheck actor then no. Fixed.